pallets/settlement/src/benchmarking.rs

```diff
@@ -127,17 +130,20 @@ fn setup_leg_and_portfolio<T: Trait>(
     index: u32,
     legs: &mut Vec<Leg<T::Balance>>,
     sender_portfolios: &mut Vec<PortfolioId>,
+    receiver_portfolios: &mut Vec<PortfolioId>,
 ) {
-    let ticker = Ticker::try_from(vec![b'A'; index as usize].as_slice()).unwrap();
-    let portfolio_from = generate_portfolio::<T>("", index, 100, from_did);
+    let variance = if index == 0 { 1 } else { index };
```

Comment:
I don't think we should be setting variance like this. Instead, do `variance = index + 1` or even better, call this function with values greater than 1.